strawberryfields/configuration.py

```diff
@@ -111,7 +113,29 @@ def create_config(authentication_token=None, **kwargs):
     return config
 
 
-def get_config_filepath(filename="config.toml"):
+def delete_config(filename="config.toml", directory=None):
+    """Delete a configuration file
+
+    Keyword Args:
+        filename (str): the configuration file to delete
+        directory (str): the directory of the configuration file if None, use
+            default directory
+    """
+    if directory is None:
+        file_path = get_default_config_path(filename)
+    else:
+        file_path = os.path.join(directory, filename)
+
+    os.remove(file_path)
+
+
+def reset_config():
+    """Delete all active configuration files"""
+    for config in get_active_configs():
+        delete_config(os.path.basename(config), os.path.dirname(config))
+
+
+def get_default_config_path(filename="config.toml"):
```

Comment:
The renaming here was done since a call to `get_config_filepath(filename)` might indicate that this function will find the `filepath` of `filename`.

Cool, `find_config_file` sounds great! :blush: